examples/sveltekit/src/routes/+layout.ts

```diff
@@ -0,0 +1,2 @@
+// export const ssr = false
+export const prerender = true
```

Comment:
```suggestion
// export const ssr = false
// export const csr = false
export const prerender = true
```

We might want to add the `csr` option as a hint. It also works great!